notebooks-src/notebooks/R Examples/Comparative Metagenomics.ipynb

```diff
@@ -0,0 +1,1339 @@
+    "      <li>Second plots: glm regression coefficients + significance + AU-ROCs for the statistical test.</li>\n",
+    "      <li>Third plots (Q-Q plot, boxplot and barplots): Original confounder check descriptive stat plots. The function produces plot for each meta-variable. When the distributions looks very similar for both controls and cases, it is unlikely that such variable would confound the analyses.</li>\n",
+    "      <li>Fourth plots: Variance explained by label versus variance explained by metadata plots. These plots show the variance explained by the label in comparison with the variance explained by the metadata variable for each individual feature. Variables with many features in the upper left corner might be confounding the label associations.</li>\n",
+    "    </ul>\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ebdbefc-39a5-4d40-9fd8-8b8c9496fb88",
+   "metadata": {},
+   "source": [
+    "### 4.3. Generation of a predictive model: Machine learning workflow <a id='part4_3'/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cf8070c-5c9c-4de0-89b2-a18a830c954b",
+   "metadata": {},
+   "source": [
+    "Another feature of `SIAMCAT` is a versatile but easy-to-use interface for the construction of machine learning meta-analysis models on the basis of microbial markers. The steps to follow include functions for data normalization, splitting the data into cross-validation folds, training the model, and making predictions based on cross-validation instances and the trained models. Such a model can be built on one dataset and then be applied on another, similarly processed holdout dataset. This might be of interest when comparing data from two different studies testing the same condition."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4291a3bb-bc92-4762-8ec7-696ff9ee8597",
+   "metadata": {},
+   "source": [
+    "1. Data normalisation. `SIAMCAT` offers a few normalization approaches that can be useful for subsequent statistical modeling in the sense that they transform features in a way that can increase the accuracy of the resulting models. Importantly, these normalization techniques do not make use of any label information, and can thus be applied up front to the whole data set and outside of the following cross validation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a68ff86c-9439-491a-83c5-3dc87fc99fa0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "siamcat_obj = normalize.features(siamcat_obj, norm.method='log.std', norm.param=list(log.n0=1e-06, sd.min.q=0))"
```

Comment:
I think it would be a bit easier to follow some of these long lines if they were broken onto multiple lines, like
```R
siamcat_obj = normalize.features(
    siamcat_obj, 
    norm.method='log.std', 
    norm.param=list(log.n0=1e-06, sd.min.q=0)
)
Especially because the scrollbar appears and blocks the text :)
```